spring-pulsar/src/test/java/org/springframework/pulsar/core/PulsarProducerFactoryTests.java

```diff
@@ -45,12 +46,19 @@ abstract class PulsarProducerFactoryTests extends AbstractContainerBaseTests {
 	protected PulsarClient pulsarClient;
 
 	@BeforeEach
-	void setup() throws PulsarClientException {
+	void createPulsarClient() throws PulsarClientException {
 		pulsarClient = PulsarClient.builder()
 				.serviceUrl(getPulsarBrokerUrl())
 				.build();
 	}
 
+	@AfterEach
+	void closePulsarClient() throws PulsarClientException {
```

Comment:
**[UNRELATED]** I noticed this test was not cleaning up when done w/ the client.